v0/destinations/salesforce/transform.js

```diff
@@ -117,11 +121,16 @@ async function getSaleforceIdForRecord(
 ) {
   const objSearchUrl = `${authorizationData.instanceUrl}/services/data/v${SF_API_VERSION}/parameterizedSearch/?q=${identifierValue}&sobject=${objectType}&in=${identifierType}&${objectType}.fields=id`;
 
-  const objSearchResponse = await axios.get(objSearchUrl, {
+  const objSearchResponse = await httpGET(objSearchUrl, {
     headers: { Authorization: authorizationData.token }
   });
 
-  return get(objSearchResponse, "data.searchRecords.0.Id");
+  if (objSearchResponse.success === false) {
+    const error = response.response;
+    errorHandler(error, "Failed to get objSearchResponse");
```

Comment:
Lets use the errorHandler from salesforce utils here?

Reply:
`Failed to get objSearchResponse` this wont be understandable by the user, we need to provide a better understandable error message 
example: `getSaleforceIdForRecord: Unable to fetch record from Salesforce`

Reply:
Done